bee-ledger/src/workers/snapshot/download.rs

```diff
@@ -126,7 +126,14 @@ async fn gather_source_information(
 ) -> Result<SourceInformation<'_>, Error> {
     let full_header = download_snapshot_header(urls.full()).await?;
     let delta_header = if download_delta {
-        Some(download_snapshot_header(urls.delta()).await?)
+        match download_snapshot_header(urls.delta()).await {
+            Ok(header) => Some(header),
+            Err(Error::DownloadingFailed) => {
+                debug!("Downloading delta snapshot header failed: {}", urls.delta());
```

Comment:
we are doing error messages in lowercase, so I guess the same applies for debug messages

Reply:
It doesn't!

Reply:
Errors are not capitalised because they are included in another sentence, logs are not

Reply:
oh makes sense, sorry!